packages/dashboard/e2e/tests/resourceExplorer/resourceExplorer.spec.ts

```diff
@@ -54,11 +54,11 @@ test('can load configure a widget with an asset model', async ({ page }) => {
   await saveAssetModel();
 
   // select property on asset model and add to widget
-  await selectProperty('Coordinates');
+  await selectProperty('Production Rate');
   await addToWidget();
 
   // check that widget is not in empty state
   await expect(page.getByText(WIDGET_EMPTY_STATE_TEXT)).not.toBeVisible();
   // check that property is visible in legend
-  await expect(grid.gridArea().getByText('Coordinates')).toBeVisible();
```

Comment:
Could you add a test to ensure properties of types a chart does not support cannot be added? This will ensure we won't regress the behavior.